src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -88,19 +90,30 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ### Path parameters
 
-    When defining an explicit path, you may mark certain parts of the path, to have them available later.
-    Sections of the path that start with a `:` will generate a method that returns that param in the action.
+    When defining an explicit path, you may mark parts of the path with a `:`,
+    to have a method generated that returns that param in the action.
+
     ```crystal
     # src/actions/users/show.cr
     class Users::Show < BrowserAction
-      get "/users/:my_user_id" do
-        text "User has id \#{my_user_id}"
+      get "/users/:some_user_id" do
+        text "Requested user id: \#{some_user_id}"
+      end
+    end
+    ```
+    Here, any string from the request will be returned by the `some_user_id` method. So in this example if 
+    `/users/1-2-foobar` is requested `some_user_id` would return a text response of `Requested user id: 1-2-foobar`.
+
+    To ensure that the action is only called with an actually existing User::ID the class can be passed
+    to the helper in the path definition `get "/users/:user_id(User::ID)"`:
+    
+    ```crystal
+    class Users::Show < BrowserAction
+      get "/users/:user_id(User::ID)" do
```

Comment:
This part is not yet implemented (planned for future release)

Reply:
hope the collapsible works